proto/rill/admin/v1/api.proto

```diff
@@ -218,6 +219,8 @@ message GetProjectResponse {
   Project project = 1;
   Deployment production_deployment = 2;
   string jwt = 3;
+  OrganizationPermissions organization_permissions = 4;
```

Comment:
I think we should consider not returning org permissions in the `GetProject` RPC. If client isn't separately requesting org details, I don't think it'll be needed.

Also, it becomes an extra DB request, and also is another place we would need to add the outside member and/or public logic.